packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -100,22 +113,22 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
     }),
 
     /** @deprecated: use getGlobalContext instead */
-    getLoggerGlobalContext: monitor(globalContextManager.get),
-    getGlobalContext: monitor(globalContextManager.getContext),
+    getLoggerGlobalContext: monitor(() => globalContextManager.get()),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What is the intent behind this change?